### PR TITLE
improve documentation on software/modules install paths

### PR DESCRIPTION
**Pull request by boegel**
and include documentation on new `--installpath-software` and `--installpath-modules` options (cfr. https://github.com/hpcugent/easybuild-framework/pull/1258)

rendered view of updated sections:

* http://boegel-eb.readthedocs.org/en/installpath_star/Configuration.html#installpath
* http://boegel-eb.readthedocs.org/en/installpath_star/Configuration.html#prefix
* http://boegel-eb.readthedocs.org/en/installpath_star/Configuration.html#installpath-subdirs

### CODE REVIEW

**Comment by boegel** [comment]
@fgeorgatos, @wpoely86, @rjeschmi: please review

File: docs/Changelog.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@
 Changelog for EasyBuild documentation
 -------------------------------------
 
+* **release 20150407.01** (`Apr 7th 2015`):
+
+  * add link to :ref:`unit_tests` page in dedicated section at :ref:`installation` page
+    (see :ref:`install_running_unit_tests`)
+  * clarify relation between ``--installpath``, ``--prefix``, ``-subdir-*`` and ``-installpath-*`` configuration options
```
Comment (fgeorgatos) [inline docs/Changelog.rst]: -- or - , explain?

File: docs/Configuration.rst [hunk 2]
```diff
@@ -291,13 +289,15 @@ If any of these configuration settings is not provided in one way or another, Ea
...
+.. note:: The mandatory path-related options can be tweaked collectively via ``--prefix``, see :ref:`prefix` for more
+          information.
 
 .. _sourcepath:
 
 Source path (``--sourcepath``)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-*default*: ``$HOME/.local/easybuild/sources/``
+*default*: ``$HOME/.local/easybuild/sources/`` (determined via :ref:`prefix`)
```
Comment (fgeorgatos) [inline docs/Configuration.rst]: determined -> also defined ?

File: docs/Configuration.rst [hunk 3]
```diff
@@ -469,24 +511,42 @@ template value here)::
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*defaults*:
+
+* *software install path subdirectory* (``--subdir-software``): ``software``
+* *modules install path subdirectory* (``--subdir-modules``): ``modules``
+* *modules install path suffix* (``--suffix-modules-path``): ``all``
+
+The subdirectories for the software and modules install paths (relative to ``--installpath``, see :ref:`installpath`)
+can be specified using the corresponding dedicated configuration options (available since EasyBuild v1.14.0).
```
Comment (fgeorgatos) [inline docs/Configuration.rst]: if you drop 1.14.0 above, drop it here; or, probably better, just keep both

**Comment by fgeorgatos** [comment]
all good!

File: docs/Changelog.rst (hunk 1 above)
Comment (boegel) [inline docs/Changelog.rst]: should be `--`

File: docs/Configuration.rst (hunk 2 above)
Comment (boegel) [inline docs/Configuration.rst]: determined is correct imho, since it's basically `<prefix>/sources`

File: docs/Configuration.rst (hunk 3 above)
Comment (boegel) [inline docs/Configuration.rst]: I'm not dropping it anywhere, this corresponds to the line being dropped that mentions v1.14.0. I just added an additional section, so the diff is a bit weird.

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos!

**Pull request merged**